temporian/core/operators/glue.py

```diff
@@ -23,7 +23,7 @@
 from temporian.proto import core_pb2 as pb
 
 # Maximum number of arguments taken by the glue operator
-MAX_NUM_ARGUMENTS = 10
+MAX_NUM_ARGUMENTS = 30
```

Comment:
very useful if we want to glue 28 lags. I had an issue for this #88 .

Reply:
:D

I wonder if we need a limit though (I am not a fan to arbitrary limits like that).